docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -12,14 +12,11 @@ Depending on your role on the Git provider you will have different permissions o
       <th>Provider</th>
       <th>Role</th>
       <th>Join organization</th>
-      <th>Ignore issues and files</th>
-      <th>Add repository</th>
+      <th>View private repository</th>
+      <th>Ignore issues and files,<br/>configure code patterns and file extensions,<br/>manage branches</th>
```

Comment:
@mrfyda I renamed the column so that it specifies the actions involved in "changing analysis configurations". Please check the end result in the live preview.